src/julienne/julienne_test_harness_s.F90

```diff
@@ -12,7 +12,11 @@
 contains
 
     module procedure component_constructor
-      allocate(test_harness%test_fixture_, source = test_fixtures)
+#ifndef __GNUC__
+      test_harness%test_fixture_ = test_fixtures ! causes a nagfor internal compiler error
+#else
```

Comment:
I think you meant the opposite sense for that comment.

Also, we've seen other compilers (Cray) that warn on implicit allocation via assignment, so (aside from the NAG ICE) explicit allocation is probably the more portable option?

How about this:

```suggestion
#ifdef NAGFOR
      test_harness%test_fixture_ = test_fixtures ! avoid a nagfor internal compiler error
#else
```